app/main.py

```diff
@@ -2,0 +1,17 @@
+def copy_file(command: str) -> None:
+    parsed_command = command.split()
+    if (
+        len(parsed_command) != 3
+        or parsed_command[1] == parsed_command[2]
+        or parsed_command[0] != "cp"
+    ):
```

Comment:
This is a code style issue. The condition in the if statement is a bit complex and hard to read. Consider splitting it into multiple simpler conditions with early returns to improve readability. For example, you could first check if the command is 'cp', then check the length of the command, and finally check if the source and destination files are the same.